docs/en/latest/deployments/ack.md

```diff
@@ -39,7 +39,6 @@ This document explains how to install Ingress APISIX on [ali-cloud ACK](https://
 cd /path/to/apisix-helm-chart
 helm repo add bitnami https://charts.bitnami.com/bitnami
 helm repo add apisix https://charts.apiseven.com
-# Use `hem search repo apisix` to search charts about apisix
```

Comment:
Why we need to remove this?

Reply:
i didn't touched this file.

Reply:
but, there's a typo in master repo in helm so i fixed that now. :)